x/gamm/client/cli/cli_test.go

```diff
@@ -870,6 +870,7 @@ func (s *IntegrationTestSuite) TestGetCmdPool() {
 
 				resp := types.QueryPoolResponse{}
 				s.Require().NoError(clientCtx.JSONCodec.UnmarshalJSON(out.Bytes(), &resp), out.String())
+				// s.Require().NoError(clientCtx.JSONMarshaler.UnmarshalJSON(out.Bytes(), &resp), out.String())
```

Comment:
err should this just be deleted, or do we have a merge conflict?

Reply:
Deleted as of now, thanks for catching this!